chrome/rhel-navigation.json

```diff
@@ -211,6 +211,12 @@
                             "href": "/openshift/subscriptions/openshift-container",
                             "product": "Subscription Watch"
                         },
+                        {
+                            "appId": "subscription-inventory",
```

Comment:
this will have to be `subscriptionInventory`. More detail can be found here: https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/docs/chrome/docs.md#registering-new-module-app-to-chrome

Reply:
Thank you for the feedback and direction.  I've updated this id and added the module configuration per the previous comment.